### DISCUSSION TITLE
Projects (beta): table of all org issues

### DISCUSSION THREAD

**Discussion by Totktonada** [community/community#5528] [Projects and Issues]
Now we can only filter / group issues that are added manually or added using some hand crafted automation. But it would be nice to do the same over the whole organization issues.

A set of views across an organization would be a good instrument to ensure that nothing sift over hands.

**Comment by p-ahl** [comment]
We have 300 open issues and 600 closed issues. Some issues are not assigned to a project and are floating around lost. Like @Totktonada says I have to look through all 300 manually or make some automation for this. Or just a filter - show all issues that are not assigned to any project or to a specific project.

**Reply by Totktonada** [reply]
I'll add our stats: ~300 public and closed-source repositories in sum, ~4.5k open issues in them. Really hard to keep it in a good shape, so anything that allows filtering / grouping would help.

Now we store links to search pages with certain filters (for periodical triage), but that's not quite convenient and not easy to maintain. The filters are for including particular set of repositories, for filtering out issues that're already in a project or milestone, to highlight issues with a particular label.

**Comment by Totktonada** [comment]
Backreferences do not work in discussions. So, I'll mention explicitly the relevant discussion about automation: https://github.com/github/feedback/discussions/5378. It is not exactly same that I described here, but I guess there are usage scenarious, which would be possible with any of those features.

**Reply by Totktonada** [reply]
I'll add one more request around the same topic: https://github.com/github/feedback/discussions/5592 asks for automatically add issues with particular labels into a project.

**Comment by Totktonada** [comment]
Now there is a relevant roadmap issue: https://github.com/github/roadmap/issues/286

> ## Summary
>
> This will help keep projects up-to-date with new issues/PRs as they are created.
>
> Currently items need to be added manually, which works well for some use-cases but can be repetitive for projects that want to see *all* the issues from a given repository.

However the issue tells about tracking issues/PRs for a repository, not organization. But if we'll able to set a *list* of repositories to track within a project, it'll work good for us for triage purposes.

It is in the 'Future' milestome ATM.
